### PR TITLE
LPS-134450 Semver

### PR DESCRIPTION
**Pull request by seltong**

### CODE REVIEW

**Comment by seltong** [comment]
ci:test:relevant

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Comment by seltong** [comment]
ci:test:relevant

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java [hunk 1]
```diff
@@ -397,15 +480,31 @@ protected Instance testGetProcessInstancesPage_addInstance(
 			Long processId, Instance instance)
 		throws Exception {
 
+		return testGetProcessInstancesPage_addInstance(
+			new HashMap<>(), processId, instance);
+	}
+
+	protected Instance testGetProcessInstancesPage_addInstance(
+			Map<Long, List<Long>> assigneeGroupIds, Long processId,
+			Instance instance)
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java:489]: @seltong, could you put instance parameter before processId?

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java [hunk 2]
```diff
@@ -479,16 +480,33 @@ public Task addTask(
 			Assignee assignee, long companyId, Instance instance, User user)
 		throws Exception {
 
+		Map<Long, List<Long>> assigneeGroupIds = new HashMap<>();
+
+		for (long role : user.getRoleIds()) {
+			assigneeGroupIds.put(role, null);
```
Comment (MarinhoFeliphe) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/helper/WorkflowMetricsRESTTestHelper.java]: @seltong, could you change the `role` variable name to `roleId`? thx

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java (hunk 1 above)
Comment (seltong) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java:489]: The method below has the same formatting problem, right? Do I have to fix this too?
protected Instance testGetProcessInstancesPage_addInstance(
			Long processId, Instance instance)

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
@MarinhoFeliphe will apply some changes and resend. Thx @seltong

**Comment by MarinhoFeliphe** [comment]
Just started reviewing :)

**Pull request closed without merging**